### PR TITLE
Initial Webapp

### LINKED ISSUE #1: Create initial structure
Going to add go-kit into the initial structure to work from.

### PR DESCRIPTION
**Pull request by tsedgwick**
We can use this as our base structure.  We will need to probably start breaking out into more specialized packages once it becomes more complex, but it's best to start simple.

This adds the initial metrics and logging for one endpoint (/providers).  I'm not sure if we want to keep that endpoint so didn't write the swagger for it just yet.  

This resolves #1

### CODE REVIEW

File: logging.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"time"
+
```
Comment (mihiramin89) [inline logging.go:5]: is there suppose to be this empty line in the import?

Comment (tsedgwick) [reply logging.go:5]: Yes, gofmt does all the formatting.  There's supposed to be a separation from imports from the standard library and third party dependencies

**Review by mihiramin89 — APPROVED** [review]
Looks good to me

File: logging.go (hunk 1 above)
Comment (mihiramin89) [reply logging.go:5]: yup pulled it down and was able to see the formatter complain.

**Pull request merged**